modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryDataRegion.java

```diff
@@ -154,22 +132,23 @@ public VolatilePageMemory pageMemory() {
     }
 
     public ReuseList reuseList() {
-        return rowVersionFreeList();
+        return freeList;
     }
 
     /**
      * Returns version chain free list.
      *
      * @throws StorageException If the data region did not start.
      */
-    public RowVersionFreeList rowVersionFreeList() {
+    public FreeListImpl freeList() {
```

Comment:
I wonder why it returns `impl` and not the interface.
Same applies for places where we declare fields. Your reply would be enough

Reply:
Sometimes we use this freelist() as ReuseList, not FreeList. It can be changed, but decided to leave it for now